themes/default/content/blog/review-stacks/index.md

```diff
@@ -0,0 +1,105 @@
+
+### Pulumi Cloud UI
+
+It is just one click to turn on Review Stacks via the Pulumi Cloud console.
+
+![Deployment Settings for Review Stacks](settings.gif)
+
+### REST API
+
+You can programmatically configure Review Stacks and Deployment Settings at scale across thousands of projects using the [Deployments REST API](/docs/pulumi-cloud/deployments/api/#patch-settings).
+
+```
+curl -i -XPOST -H "Content-Type: application/json" -H "Authorization: token $PULUMI_ACCESS_TOKEN" \
+--location "https://api.pulumi.com/api/preview/org/project/stack/deployment/settings" \
+-d '{
+  "gitHub":{
+    "pullRequestTemplate": true
+    }
+}'
+```
+
+### Pulumi Cloud Resource Provider
+
+You can use Pulumi to manage and code review Deployment Settings and Review Stacks with the [Pulumi Cloud Provider](/registry/packages/pulumiservice).
+
+```typescript
+import * as pulumiservice from "@pulumi/pulumiservice";
+
+const deploymentSettings = new pulumiservice.DeploymentSettings("deploymentSettings", {
+	organization: pulumi.getOrganization(),
+	project: "your project",
+	stack: "your stack",
+	github: {
+		deployCommits: true,
+		previewPullRequests: true,
+		pullRequestTemplate: true,
+		repository: "pulumi/deployment-automation",
+	},
+	sourceContext: {
+		git: {
+			branch: "refs/heads/main",
+			repoDir: "pulumi-pet-shop",
+		},
+	},
+});
+```
+
+## Collaborate in the Cloud
+
+At Pulumi, our engineering team writes *a ton* of code. We have hundreds of repositories with programs managing thousands of stacks and tens of thousands of cloud resources. These tools and services power production workloads, internal tools, business intelligence and process, and more. And of course we use Pulumi to ship all of it. When you’re an engineer working in one of these repositories, making a code change is the easy part. The hard part is figuring out how to get a tight dev loop going---in particular, faithfully verifying your changes before merging and deploying them. Are there instructions in the README for running locally? A Docker Compose file? Or maybe standalone dev environments? Can I figure out what config and upstream stack references are required to stand up a fresh dev stack? And what about service dependencies? Pulumi makes it incredibly easy to build in the cloud and write prolific amounts of code, however, with that increase in development velocity comes a need to standardize the entire process by which we develop, test, review, and ship our code.
+
+We've heard similar stories from hundreds of customers. From startup teams of five to enterprises of thousands, they're all looking for the same thing - so we went ahead and built it. With Review Stacks and Pulumi Deployments, you can ship with greater velocity, confidence, and collaboration.
+
+Check out these resources and get started today:
+
+- TODO Review Stacks docs link
```

Comment:
We'll have a workshop on Review Stacks with the GitHub and Azure teams ready to add here once this issue ships: https://github.com/pulumi/marketing/issues/763